deploy/gke/redaction/README.md

```diff
@@ -0,0 +1,86 @@
+## Redacting span attributes with the OpenTelemetry collector
+
+This is a GKE-specific guide for using the collector's [redaction processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/redactionprocessor#redaction-processor). See the upstream documentation for the config reference.
+
+If this is the first example you are trying out, follow the [Setup](../setup.md) instructions to
+complete the prerequisites.
+
+See the [Troubleshooting](../troubleshooting.md) guide for more information if you encounter issues.
+
+### Set up namespace
+
+Create a namespace in your cluster to run the collector:
+
+```
+export OTEL_NAMESPACE=otel-collector
+kubectl create namespace $OTEL_NAMESPACE
+```
+
+### Create ConfigMaps
+
+The [`otel-config.yaml`](otel-config.yaml) file contains a sample OpenTelemetry Collector config that is
+prepopulated with some of the receivers, exporters, and processors included in this project. Edit it to
+your desired [configuration](https://opentelemetry.io/docs/collector/configuration/) and create a ConfigMap
+from it in the namespace you created above:
+
+```
+kubectl create configmap otel-config --from-file=./otel-config.yaml -n $OTEL_NAMESPACE
+```
+
+The [`testdata.json`](testdata.json`) file contains OpenTelemetry Line Protocol (OTLP) data with traces
+that have credit card numbers as attributes.  We use this data as input to show how to mask the credit
+card numbers.
+
+```
+kubectl create configmap testdata --from-file=./testdata.json -n $OTEL_NAMESPACE
+```
+
+
+### Create the Deployment
+
+When you built your collector image, the file [`manifest.yaml`](manifest.yaml)
+was automatically updated to reference your image name. Create this manifest in your cluster with:
+
+```
+cd deploy/gke/simple/
+kubectl apply -f manifest.yaml -n $OTEL_NAMESPACE
+```
+
+### Try it out!
+
+Take a look at [`testdata.json`](testdata.json`).  It contains an attribute
+`credit.card.number=visa credit card number: 4111111111111111` which matches
+our redaction regexp filter in [`otel-config.yaml`](otel-config.yaml):
+`4[0-9]{12}(?:[0-9]{3})?`.
+
+Check the collector deployment's logs to see the attribute now that it has been redacted:
+
+```
+kubectl logs deploy/otel-collector -n $OTEL_NAMESPACE | grep "visa credit card number"
```

Comment:
a sample of the actual redacted lines would be nice here to compare the user's output against

Reply:
removed this section